test/util/e2e.go

```diff
@@ -414,24 +415,28 @@ func WaitForKubeSystemControllersAvailability(ctx context.Context, k8sClient cli
 	ginkgo.By(fmt.Sprintf("Waiting for deployment %q to be available", deployKey.Name))
 	waitForDeploymentAvailability(ctx, k8sClient, deployKey)
 
-	for _, ds := range []string{
-		"kindnet",
-		"kube-proxy",
-	} {
-		dsKey := types.NamespacedName{Namespace: ns, Name: ds}
-		ginkgo.By(fmt.Sprintf("Waiting for daemonset %q to be available", ds))
-		waitForDaemonSetAvailability(ctx, k8sClient, dsKey)
-	}
+	for pass := range 2 {
```

Comment:
I have also added another pass over availability of kube-system Pods, because I think it is possible that we see some pod say kube-controller-manager as Ready, but it becomes NotReady becasue api-server is down, and only later we confirm api-server to be Ready.